tests/phpunit/JsonDecoderTest.php

```diff
@@ -25,7 +26,8 @@ public function testValidJson(string $script, array $expectedDocuments): void
     {
         $parsedDocuments = [];
         $process = $this->createScriptProcess($script);
-        $this->jsonDecoder->processStream($process->stdout, function (array &$document) use (&$parsedDocuments): void {
+        $jsonStream = $process->pipes[ProcessFactory::JSON_STREAM_FD];
```

Comment:
A v PHP potom na tieto JSON dokumenty citame.

Reply:
(tento riadok som este mierne upravoval, preto pise `outdated`, neviem to uz editovat).